pyaedt/emit_core/Couplings.py

```diff
@@ -9,12 +9,12 @@ class CouplingsEmit(object):
     """Provides for interaction with the EMIT ```coupling`` folder,
```

Comment:
```suggestion
    """Provides for interaction with the EMIT ```coupling`` folder.
    
```

Reply:
When it's not referring to code, yes it's the product name and typically all caps.